java/clients/src/main/java/sleeper/clients/admin/IngestStatusReportScreen.java

```diff
@@ -0,0 +1,130 @@
+            ).run();
+        }
+    }
+
+    private void chooseArgsForIngestJobStatusReport(String instanceId) throws InterruptedException {
+        Optional<TableProperties> tableOpt = tableSelectHelper.chooseTableOrReturnToMain(instanceId);
+        if (tableOpt.isPresent()) {
+            String tableName = tableOpt.get().get(TableProperty.TABLE_NAME);
+            consoleHelper.chooseOptionUntilValid("Which query type would you like to use",
+                    new MenuOption("All", () ->
+                            runIngestJobStatusReport(instanceId, tableName, JobQuery.Type.ALL)),
+                    new MenuOption("Unfinished", () ->
+                            runIngestJobStatusReport(instanceId, tableName, JobQuery.Type.UNFINISHED)),
+                    new MenuOption("Detailed", () ->
+                            runIngestJobStatusReport(instanceId, tableName, JobQuery.Type.DETAILED,
+                                    promptForJobId(in))),
+                    new MenuOption("Range", () ->
+                            runIngestJobStatusReport(instanceId, tableName, JobQuery.Type.RANGE,
+                                    promptForRange(in)))
+            ).run();
+        }
+    }
+
+    private void chooseArgsForIngestTaskStatusReport(String instanceId) throws InterruptedException {
+        consoleHelper.chooseOptionUntilValid("Which query type would you like to use",
+                new MenuOption("All", () ->
+                        runIngestTaskStatusReport(instanceId, IngestTaskQuery.ALL)),
+                new MenuOption("Unfinished", () ->
+                        runIngestTaskStatusReport(instanceId, IngestTaskQuery.UNFINISHED))
+        ).run();
+    }
+
+    private IngestJobStatusReportArguments.Builder argsBuilder(String instanceId, String tableName, JobQuery.Type queryType) {
+        return IngestJobStatusReportArguments.builder()
+                .instanceId(instanceId).tableName(tableName)
+                .queryType(queryType)
+                .reporter(new StandardIngestJobStatusReporter(out.printStream()));
+    }
+
+    private void runIngestJobStatusReport(String instanceId, String tableName, JobQuery.Type queryType, String queryArgs) {
+        runIngestJobStatusReport(instanceId, argsBuilder(instanceId, tableName, queryType)
+                .queryParameters(queryArgs).build());
+    }
+
+    private void runIngestJobStatusReport(String instanceId, String tableName, JobQuery.Type queryType) {
+        runIngestJobStatusReport(instanceId, argsBuilder(instanceId, tableName, queryType).build());
+    }
+
+    private void runIngestJobStatusReport(String instanceId, IngestJobStatusReportArguments args) {
+        InstanceProperties instanceProperties = store.loadInstanceProperties(instanceId);
+        new IngestJobStatusReport(store.loadIngestJobStatusStore(instanceId), args,
+                store.getSqsClient(), instanceProperties.get(INGEST_JOB_QUEUE_URL)).run();
```

Comment:
We could pass the instance properties into IngestJobStatusReport to ensure the same property is used in the main method.